tests/e2e/specs/pacbio_plates_view.js

```diff
@@ -0,0 +1,16 @@
+describe('Pacbio plates view', () => {
```

Comment:
It would also be worth testing that the plate is there and has a filled well.

Reply:
👍